src/appmixer/line/auth.js

```
@@ -0,0 +1,44 @@
+        },
+
+        async requestProfileInfo(context) {
+            // Use the LINE profile endpoint to verify the token and get user info
+            // We use a dummy userId (U4af4980629...) as per LINE docs for token validation
+            // But for real profile, userId is required. For validation, we can call /v2/bot/info
+            const url = 'https://api.line.me/v2/bot/info';
+            const headers = {
+                'Authorization': `Bearer ${context.channelAccessToken}`
+            };
+            const { data } = await context.httpRequest({ url, method: 'GET', headers });
+            if (data && data.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)